src/components/tabs/js/tabsDirective.js

```diff
@@ -146,6 +146,7 @@ function MdTabs ($$mdSvgRegistry) {
             '<md-icon md-svg-src="'+ $$mdSvgRegistry.mdTabsArrow +'"></md-icon> ' +
           '</md-next-button> ' +
           '<md-tabs-canvas ' +
+              'tabindex="0" ' +
```

Comment:
Please go back to what the old commit did `'tabindex="{{ $mdTabsCtrl.hasFocus ? -1 : 0 }}" ' +`. This allows for shift+tab to escape from the tabs control. With it staying `0` all of the time, shift+tab gets stuck and can't get out of the tabs control.